src/main/java/com/dlsc/jfxcentral/data/model/ModelObject.java

```diff
@@ -118,6 +116,14 @@ public void setTags(String tags) {
         this.tags = tags;
     }
 
+    public String getCanonical() {
```

Comment:
Please add "Url" to the method / field so that it is more descriptive of what we are expecting.